app/build.gradle

```diff
@@ -58,7 +58,7 @@ android {
             storeFile file("../debug.keystore")
         }
         release {
-            storeFile file(getKey("SIGNING_KEYFILE_PATH", "../gnosis-upload.jks"))
```

Comment:
Just to clarify: This just changes the environment variable to look for the keytore file name in

Reply:
yes